src/App.tsx

```diff
@@ -1,26 +1,169 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import { FC, useState, useEffect, useMemo, useCallback } from 'react';
+import {
+  USER_ID,
+  getTodos,
+  addTodo,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorMessage } from './types/ErrorMessage';
+import { StatusFilter } from './types/StatusFilter';
+import { TodosHeader } from './components/TodosHeader';
+import { TodoList } from './components/TodoList';
+import { TodosFooter } from './components/TodosFooter';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessage | null>(null);
```

Comment:
u don't need `null` for this state.
`ErrorMessage.default = ''`
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorMessage>(ErrorMessage.Default);
```